src/web/styles.scss

```diff
@@ -136,3 +136,32 @@ body {
 .editor.editor-disabled {
   cursor: not-allowed;
 }
+
+@keyframes myExpand {
+  from {
+    opacity: 0;
+    max-height: 0;
+  }
+
+  to {
+    opacity: 1;
+    max-height: 800px;
+  }
+}
+
+.collapse {
+  transition: .35s ease-in-out;
+  opacity: 0;
+  visibility: collapse;
+  max-height: 0;
+  display: block !important;
+  overflow: hidden;
+
+  &.show {
+    opacity: 1;
+    max-height: 99999999px;
+    overflow: visible;
+    visibility: visible;
+    animation: myExpand .35s ease-in-out;
+  }
+}
```

Comment:
This part is the main part of the PR, adapted mainly from https://stackoverflow.com/a/8331169 and https://stackoverflow.com/a/40143101 

`max-height: 99999999px;` because our content is variable height.
`display: block !important;` to override `display: none` of the default `collapse` class. Otherwise animation would not play.